### PR TITLE
The shipping news: Add new NEWS and cleanup old NEWS.

### PR DESCRIPTION
**Pull request by un1versal**
If anyone wants to add something speak now please,

The HTML has been checked with http://codebeautify.org/htmlviewer/

### CODE REVIEW

File: news/latest/2016.01.02.08.06.1.story [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+It is our hope our users wont stumble into quite so many typos (even if the odd one slipped through), but also our hard working translators that will hopefully not have to work around issues with core strings in their translations anymore.
+Many string consistencies have been addressed, its possible some have managed to slip through though, but we hope to squash them all eventually.<br>
+We certainly would appreciate that our translators/users report any issues at our <a href="https://github.com/kvirc/KVIrc/issues">KVIrc issue tracker</a>, like broken syntax a.k.a. broken English or any other issue, so we can instead of working around it in translations, fix t in core instead.
+<br></p><p>
+While Im personally not a fan of stats and the stats bellow at are missing some data thats not accounted for, the move to GitHub makes collating these changes a much simpler process; here are some stats you may find interesting:
+</p><li><a href="https://github.com/kvirc/KVIrc/commit/6c7203936f98e782c23e365ebc368919c4623004">6000 commits</a> have made its way into Master since moving to GitHub (100% accurate at time or writing)
```
Comment (DarthGandalf) [inline news/latest/2016.01.02.08.06.1.story]: Lies
It's more like 911 commits.

Comment (un1versal) [inline news/latest/2016.01.02.08.06.1.story]: mmm not, its 6000, it says so in github

Comment (DarthGandalf) [inline news/latest/2016.01.02.08.06.1.story]: You're confusing 2 different numbers.
Since beginning of times it's 6000.
Since moving to Github it's 911.

Comment (un1versal) [inline news/latest/2016.01.02.08.06.1.story]: mmm Ill look into that and update this bit. thx dindt think that was counted but yea.

Comment (DarthGandalf) [inline news/latest/2016.01.02.08.06.1.story]: And 739 if to ignore merge commits.

Comment (un1versal) [inline news/latest/2016.01.02.08.06.1.story]: have chnaged that now, please check..

**Comment by un1versal** [comment]
~~Im adding a few thing more, so dont merge yet...~~

If you guys want stuff adding, please let me know what you'd like to see and some details about it... And Ill work it in.

But ... We should do more write ups more often...

IF anyone knows any nifty tricks to grab a full changelog for all the changes that would also be nice... That doesnt involve much manual work. WOuld be nice to have...
- Changes since moving to GitHub
- Changes since 4.2.0

**Comment by un1versal** [comment]
OK ready to be reviewed and if none has anything else please do the honors.

**Pull request merged**